src/index/VocabularyImpl.h

```diff
@@ -146,25 +146,12 @@ bool Vocabulary<S>::isExternalizedLiteral(const string& word) {
 
 // _____________________________________________________________________________
 template <class S>
-template <bool isEntity>
 bool Vocabulary<S>::shouldBeExternalized(const string& word) const {
-  if constexpr (isEntity) {
```

Comment:
As there is currently no use for this compile-time check and I asume that the string operations will always dominate the runtime it is fine for me to remove this (I currently don't remember the initial reason for this)